tasks/install_shell.sh

```diff
@@ -132,12 +132,13 @@ elif test -f "/usr/bin/sw_vers"; then
 
   major_version=`echo $platform_version | cut -d. -f1,2`
   case $major_version in
-    "10.6") platform_version="10.6" ;;
-    "10.7"|"10.8"|"10.9") platform_version="10.7" ;;
-    *) echo "No builds for platform: $major_version"
+    "10.6"|"10.7"|"10.8"|"10.9"|"10.10") echo "No builds for platform: $major_version"
        report_bug
```

Comment:
We can probably leave this out

Reply:
Leave out report_bug? Or the version check in general?